openweathermap-ios/openweathermap-ios/Maps/ViewControllers/MapViewController.swift

```diff
@@ -11,6 +11,10 @@ final class MapViewController: UIViewController {
 
     @IBOutlet private weak var mapView: MKMapView!
     private let locationManager = CLLocationManager()
+    private var weatherManager = WeatherManager()
```

Comment:
Are you planning to set a different manager here? If not, consider letting it as constant.

Reply:
When I put "weatherManager.delegate = self" in viewDidLoad I need to use var. Have any alternative?

Reply:
It is related to https://github.com/lucasmontano/openweathermap-ios/pull/9/files#r405690436